EIPS/eip-4973.md

```diff
@@ -185,7 +185,25 @@ You can find an implementation of this standard in [../assets/eip-4973](../asset
 
 ## Security Considerations
 
-There are no security considerations related directly to the implementation of this standard.
+### Privacy Considerations
+
+#### Storing Personally Identifiable Information (PII) on-chain presents privacy risks
```

Comment:
before we mention privacy risk, we'd have to define privacy as e.g. "violation of contextual integrity." But then again, I think it must not be within the scope of eip4973 but an informal EIP document.